src/olympia/shelves/tests/test_serializers.py

```diff
@@ -149,6 +152,7 @@ def test_basic(self):
         data = self.serialize(self.search_rec_ext)
         assert data['title'] == {'en-US': 'Recommended extensions'}
         assert data['endpoint'] == 'search'
+        assert data['addon_type'] == 1
```

Comment:
but the serialized result should be a string - it's okay to literally test for `'extension'` here (no need to do any lookup for the string constant for an extension)

Reply:
And the same elsewhere in the file - Shelf model should be using `ADDON_EXTENSION` (or whatever); testing `data['addon_type']` should be checking for the literal string value.